src/libraries/System.Data.OleDb/src/System.Data.OleDb.csproj

```diff
@@ -1,9 +1,9 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFrameworks>$(NetCoreAppCurrent)-windows;$(NetCoreAppCurrent);$(NetCoreAppMinimum)-windows;$(NetCoreAppMinimum);netstandard2.0;$(NetFrameworkMinimum)</TargetFrameworks>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <!-- Supress CA2249: Consider using String.Contains instead of String.IndexOf to avoid ifdefs. -->
-    <NoWarn>$(NoWarn);CA2249</NoWarn>
+    <NoWarn>$(NoWarn);CA2249;CA1420</NoWarn>
```

Comment:
@jkoritzinsky I was not able to figure out how to fix `warning CA1420: Delegates with managed types as parameters or the return type require runtime marshalling to be enabled in the assembly where the delegate is defined.`. Please let me know how to handle this

21 warnings in total here and in `System.Drawing.Common`

Reply:
My commits in https://github.com/dotnet/runtime/pull/69709 have the fixes for the new analyzer warnings.

Reply:
Thanks, will rebase after that merged